### ISSUE TITLE
Add a Magisk Module to setup microg

### ISSUE THREAD

**Issue by Mannshoch** [microg/GmsCore#1314]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I have to check internet again how to get microg work and how to setup e.g. UnifiedNlp, 

**Describe the solution you'd like**

- A Official MicroG Module in Magisk.
- It describe what it do and what you could do.
- After activate this module you get a setup dialog that ask you about the wished features you wish and may some settings for this feature.
- After that this modlule download all needed files and ask for reboot to setup them in twrp/lineageOS recovery and add all needed apk

**Describe alternatives you've considered**
 Visit several https://www.xda-developers.com and https://microg.org/download.html and do it my self

**Comment by lss4** [comment]
You can already check out NanoDroid ([official site](https://nanolx.org/nanolx/nanodroid), [XDA](https://forum.xda-developers.com/t/module-system-nanodroid-23-0-1-20201029-microg-pseudo-debloat-f-droid-apps.3584928/)). You can setup microG and some other open source stores systemlessly with it. It also has support for Play Store with working IAP over microG.

It may still be a good idea for having an official, pure Magisk module of microG as NanoDroid includes a lot of stuffs that you need to carefully configure the installation to suit your need.

**Comment by Mannshoch** [comment]
Nanodroid seams very interesting. But It contian a bit to much. ( I don't see the "Nano" in this tool)

I simply vote here for a microg only installer. No f-droid and no other stuff but the possibility to get a complete microg experience.

**Comment by chris42** [comment]
Check the documentation. AFAIK you can configure what is installed. @Nanolx <--- might where the name came from ;-D

**Comment by Mannshoch** [comment]
I'll look into it, thanks.  

(still vote for a tool from microg 😉 )

**Comment by pflanzenandi** [comment]
Another project already exists:

https://github.com/nift4/microg_installer_revived

**Comment by Mannshoch** [comment]
Yes, but you are the pro's here. I don't like it to collect hundreds of different apps from a hundred different devs, which do root things on my phone.

**Comment by chris42** [comment]
Not sure if Marvin is a pro in magisk modules, nor if it is on his roadmap. That is the advantage of the community, that everyone can do what he can do best. If you want a recommendation: Nanodroid is around for quite a while and @Nanolx quite responsive to questions and updates.

Of course, you can still ask for the magisk module, just not make near future plans with it, until Marvin has said something. :-)

**Comment by Mannshoch** [comment]
OK 👍

**Comment by Nanolx** [comment]
AFAIK mar-v-in said an installer *may* become official, when it

a) be ableto not just grab pre-build APKs, but also be able to build from source code aswell
b) does not include anything closed source

while b) is rather easily achieved, a) requires more work, more than most are willing to do.

Also many ROMs now have built-in microG support or even microG bundled (LOS4microG, /e/OS).
